.github/workflows/tag.yml

```diff
@@ -8,6 +8,9 @@ on:
     paths:
       - version.txt
 
+permissions:
+  contents: read
```

Comment:
This one needs `write`, because it has to `git push` the new tag

Reply:
This change is just to set the minimum default access as read for all jobs in the workflow. I think it will not affect the existing job since we already have a job specific write permission for that.

Reply:
Oh I didn't notice the job's config, thanks.